extensions/amp-bind/0.1/bind-impl.js

```diff
@@ -490,6 +490,18 @@ export class Bind {
     });
   }
 
+  /**
+   * Evaluates bind expression and then updates given elements (and their subtrees).
+   *
+   * @param {!Array<!Element>} elements
+   * @return {!Promise}$
+   */
+  constrainedEvaluation(elements) {
+    return this.evaluate_().then(results =>
+      this.applyElements_(results, elements)
+    );
+  }
```

Comment:
Is this method used?

Reply:
the whole PR is pretty stale at this point. will get it in proper form today